src/app/config/urls.ts

```diff
@@ -80,7 +80,9 @@ export const jobTypeToPath = (type: JobTypes, result?: boolean) => {
     case JobTypes.BLAST:
       return LocationToPath[result ? Location.BlastResult : Location.Blast];
     case JobTypes.IDMAP:
```

Comment:
If we are removing the name of id mapping, do change the job type too to avoid having places with IDmapping or related terms, and others saying upload list

Reply:
Good point, missed that.